prow/cluster/build/400-boskos-janitor.yaml

```diff
@@ -47,11 +47,11 @@ spec:
           readOnly: true
         resources:
           requests:
-            cpu: 200m
-            memory: 20Mi
+            cpu: 2000m
```

Comment:
This is likely more than enough. For now let's keep the requests and only bump up the limits, and see how it goes?

```
requests:
    cpu: 200m
    memory: 20Mi
limits:
    cpu: 2000m
    memory: 2560Mi
```

Reply:
Fixed